frontend/src/ui-components/AcmSearchInput/AcmSearchInput.tsx

```diff
@@ -191,7 +213,44 @@ export function AcmSearchInput(props: Readonly<AcmSearchInputProps>) {
                         {getAvailableColumnNames()}
                       </AcmSelect>
                     </FlexItem>
-                    <FlexItem style={{ width: '35%' }}>
+                    <FlexItem style={{ minWidth: '13%' }}>
+                      <div
+                        onMouseEnter={() => {
+                          const newIsHoveredArray = { ...isHovered }
+                          newIsHoveredArray[index] = true
+                          setIsHovered(newIsHoveredArray)
+                        }}
+                        onMouseLeave={() => {
+                          const newIsHoveredArray = { ...isHovered }
+                          newIsHoveredArray[index] = false
+                          setIsHovered(newIsHoveredArray)
+                        }}
```

Comment:
Is the best way to handle this (they are helping me render the tooltip onMouseHover), to use a window listener?

Sonar is describes my method here as [using a non-native interactive element](https://sonarcloud.io/project/issues?sinceLeakPeriod=true&issueStatuses=OPEN%2CCONFIRMED&pullRequest=3902&id=open-cluster-management_console&open=AZItsWDZw-ADpA4W3s_k).